src/views/Login.js

```diff
@@ -70,6 +72,20 @@ export default function Login() {
 
   const [deducing, setDeducing] = useState(false);
 
+  // again, using breakpoint to decide if we're on mobile or not
+  // see ActivateCode for more details
+  const hardwareAllowed = useBreakpoints([false, true, true]);
```

Comment:
As per my comment in that asana task, unsure if disallowing hardware wallets on mobile is the right thing to do. Not sure what we gain from disallowing it. Hardware wallets might be the only sane/safe way of doing crypto-things on mobile! cc @jyng

Reply:
trezor at least definitely doesn't work on mobile, but ledger might?

Reply:
I'd argue that regardless of whether they are actually supported or not, we should still display them on mobile? Cultivate user awareness, and also future-proofs. But maybe Design feels strongly about likely-unusable things being too ugly in that context?